### PR TITLE
Add reset user home chart

### PR DESCRIPTION
**Pull request by tomskelley-gjs**
https://trello.com/c/CxKqlUpN/654-ripley-button-resets-a-users-conda-environment-etc

### CODE REVIEW

**Comment by damacus** [comment]
I'm missing something with the design here (possibly?), what's the use of the Ingress controller here?

File: charts/reset-user-home/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Rest user home environment
```
Comment (andylightfoot) [inline charts/reset-user-home/Chart.yaml:3]: Small typo with Rest vs Reset

File: charts/reset-user-home/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{{- if .Values.ingress.enabled -}}
```
Comment (andylightfoot) [inline charts/reset-user-home/templates/ingress.yaml:1]: I don't think you need an ingress for this

File: charts/reset-user-home/templates/service.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```
Comment (andylightfoot) [inline charts/reset-user-home/templates/service.yaml:1]: Not sure you need a service for this either

File: charts/reset-user-home/values.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+image:
+  repository: "quay.io/mojanalytics/airflow"
+  tag: "1.10.6-1"
+  pullPolicy: "IfNotPresent"
+  
+nameOverride: ""
+fullnameOverride: ""
+
+service:
```
Comment (andylightfoot) [inline charts/reset-user-home/values.yaml:17]: Not sure this needs a service?

File: charts/reset-user-home/values.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  type: ClusterIP
+  port: 80
+
+ingress:
```
Comment (andylightfoot) [inline charts/reset-user-home/values.yaml:21]: Not sure this needs an ingress?

File: charts/reset-user-home/Chart.yaml (hunk 1 above)
Comment (tomskelley-gjs) [reply charts/reset-user-home/Chart.yaml]: Fixed

File: charts/reset-user-home/templates/ingress.yaml (hunk 2 above)
Comment (tomskelley-gjs) [reply charts/reset-user-home/templates/ingress.yaml]: Removed - was added by the template - I just didn't remove it

File: charts/reset-user-home/templates/service.yaml (hunk 3 above)
Comment (tomskelley-gjs) [reply charts/reset-user-home/templates/service.yaml]: Removed - was also added by the template - I just didn't remove it

File: charts/reset-user-home/values.yaml (hunk 4 above)
Comment (tomskelley-gjs) [reply charts/reset-user-home/values.yaml:17]: Removed

File: charts/reset-user-home/values.yaml (hunk 5 above)
Comment (tomskelley-gjs) [reply charts/reset-user-home/values.yaml:21]: Removed

**Pull request merged**